conans/client/build/compiler_flags.py

```diff
@@ -250,6 +250,11 @@ def format_libraries(libraries, settings):
             if not library.endswith(".lib"):
                 library += ".lib"
             result.append(library)
+        elif library.startswith("lib") and library.endswith(".a"):
```

Comment:
This is the legacy build systems integrations support, that are already removed in 2.0. Better not touch them.